src/main/java/frc/robot/commands/CDSReverseCommand.java

```diff
@@ -5,22 +5,27 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
-import frc.robot.subsystems.CDSSubsystem; 
+import frc.robot.subsystems.CDSSubsystem;
+import frc.robot.subsystems.ShooterSubsystem; 
 
 public class CDSReverseCommand extends CommandBase {
   /** Creates a new IntakeForwardCommand. */
   private final CDSSubsystem mCDSSubsystem;
+  private final ShooterSubsystem mShooterSubsystem;
   
-  public CDSReverseCommand(CDSSubsystem CDSSubsystem) {
+  public CDSReverseCommand(CDSSubsystem CDSSubsystem, ShooterSubsystem shooterSubsystem) {
     // Use addRequirements() here to declare subsystem dependencies.
-    addRequirements(CDSSubsystem); 
+    addRequirements(CDSSubsystem);
+    addRequirements(shooterSubsystem);
+    mShooterSubsystem = shooterSubsystem;
     mCDSSubsystem = CDSSubsystem;
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
     mCDSSubsystem.CDSBeltWheelControllerToggle(true);
+    mShooterSubsystem.runCargo(true, false);
```

Comment:
The overall command here reverses the CDS. Do we want the cargo stopper to do anything?

Reply:
We do, actually. This is because there is a possibility the driver forced the ball too far into the shooter, and needs to reverse it.